navigation/src/androidMain/kotlin/com/freeletics/khonshu/navigation/internal/StackSnapshot.kt

```diff
@@ -22,8 +22,11 @@ public class StackSnapshot internal constructor(
     internal val current: StackEntry<*>
         get() = entries.last()
 
+    internal val previous: StackEntry<*>
+        get() = entries.getOrNull(entries.size - 2) ?: startStackRootEntry
```

Comment:
What do you think of using `lastIndex - 1` instead of `size - 2` to get index before the last?

```suggestion
        get() = entries.getOrNull(entries.lastIndex - 1) ?: startStackRootEntry
```